mypyc/genops.py

```diff
@@ -96,11 +97,44 @@ def build_ir(modules: List[MypyFile],
 
     # Compute vtables.
     for _, cdef in classes:
-        mapper.type_to_ir[cdef.info].compute_vtable()
+        compute_vtable(mapper.type_to_ir[cdef.info])
 
     return result
 
 
+def compute_vtable(cls: ClassIR) -> None:
+    """Compute the vtable structure for a class."""
+    if cls.vtable is not None: return
```

Comment:
Style nit: move return statement to a separate line (I wonder if we can add a lint rule for this).